worker/pkg/workflows/datasync/activities/activities.go

```diff
@@ -544,7 +550,7 @@ func getMysqlDsn(
 	}
 }
 
-func (b *benthosBuilder) buildProcessorConfig(ctx context.Context, cols []*mgmtv1alpha1.JobMapping) (*neosync_benthos.ProcessorConfig, error) {
+func (b *benthosBuilder) buildProcessorConfig(ctx context.Context, cols []*mgmtv1alpha1.JobMapping) ([]*neosync_benthos.ProcessorConfig, error) {
```

Comment:
This is returning a slice now, can the method be updated to `buildProcessorConfigs`?

Reply:
udpated